stdlib/2and3/optparse.pyi

```diff
@@ -187,8 +187,8 @@ class OptionParser(OptionContainer):
     def _get_args(self, args: Iterable) -> List[Any]: ...
     def _init_parsing_state(self) -> None: ...
     def _match_long_opt(self, opt: _Text) -> _Text: ...
-    def _populate_option_list(self, option_list: List[Option], add_help: bool = ...) -> None: ...
-    def _process_args(self, largs: List[_Text], rargs: List, values: Values) -> None: ...
+    def _populate_option_list(self, option_list: Sequence[Option], add_help: bool = ...) -> None: ...
```

Comment:
This is called with `option_list` from `__init__` as an argument, so `Iterable[Option]` if you are changing the others.